### DISCUSSION TITLE
如何在规则集中判断「前台窗口为桌面」？

### DISCUSSION THREAD

**Discussion by DryIce-cc** [ClassIsland/ClassIsland#407] [Q&A] [answered]
在外部工具中，我获取到「桌面」窗口的信息如下：
- 标题：FolderView
- 类名：SysListView32
- 进程：explorer.exe

但我将其输入到规则中，规则集判定不生效。
有谁知道「桌面」窗口的各项信息？

**Comment by HelloWRC** [comment, marked as answer]
那个窗口类名的规则只判定顶层窗口的类名，桌面顶层窗口的类名应该是 Progman
